tools/prove_wrapper

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
By default for every bash script I recommend `set -euo pipefail` to prevent nasty surprises

Reply:
Done